frontend/src/MyProfile/ProfileModal.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useContext } from 'react';
+import './ProfileModal.css';
+import { UserContext } from '../UserContext';
+
+export default function ProfileModal({ isOpen, onClose, handleSubmitPatientInfo }) {
+  const { user } = useContext(UserContext);
+  const [formData, setFormData] = useState({
+    firstname: '',
+    lastname: '',
+    place_of_birth: '',
+    date_of_birth: '',
+    sex: '',
+    height: '',
+    weight: '',
+    occupation: '',
+    address: '',
+    phone: '',
+    complaint: '',
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormData((prevData) => ({ ...prevData, [name]: value }));
+  };
+
+  if (!isOpen) return null;
+
+  return (
+    <div className="modal-overlay">
+      <div className="modal-content">
+        <form onSubmit={(e) => handleSubmitPatientInfo(e, formData)}>
+          <p>CREATE PATIENT PROFILE</p>
```

Comment:
nit: +1, maybe we should use ``Create patient profile`` instead, thoughts?

Reply:
I've effected the changes  and commited them.